packages/cli/src/telemetry/index.ts

```diff
@@ -125,6 +126,7 @@ export class Telemetry {
 			) {
 				// errored exec
 				void this.track('Workflow execution errored', properties);
+				// TODO UM: check if user id must be provided above. Owner ID?
```

Comment:
out of scope for this PR?

Reply:
Comment doesn't make sense anymore, it was left as a reminder. Thanks for pointing it out!